microprofile/pom.xml

```diff
@@ -46,6 +46,29 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>io.fabric8</groupId>
+        <artifactId>fabric8-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <goals>
+              <goal>resource</goal>
```

Comment:
I thought it's idiomatic to define both `resource` and `build` goals here, and have users only invoke `mvn fabric8:deploy` instead of `mvn fabric8:build fabric8:deploy`?